src/web/routes/application/confirm/confirm.js

```diff
@@ -31,8 +31,11 @@ const getConfirmPage = (req, res, next) => {
 
     return res.render('confirm', {
       title: getTitle(req),
-      subTitle: req.t('confirm.subTitle', { totalVoucherValue: toPounds(totalVoucherValueInPence) }),
-      totalVoucherValueForFourWeeks: toPounds(totalVoucherValueInPence * 4)
+      subTitle: req.t('confirm.subTitle',
+        {
+          totalVoucherValue: toPounds(totalVoucherValueInPence),
+          totalVoucherValueForFourWeeks: toPounds(totalVoucherValueInPence * 4)
```

Comment:
This might not be the correct value :(
If they have applied just before a child turns 4 this won't be correct. It's not been introduced as part of this commit though, and the communication around such a  scenario would be really tricky, so let's leave it for now...

Reply:
Yes, that sounds like an edge case which we probably wouldn't want to complicate this page with. Might be worthwhile raising with Karen and/or Sam and see whether this needs to be documented somewhere like the Business Rules document?